pkg/state/app/status.go

```diff
@@ -0,0 +1,28 @@
+package app
+
+// Status app status
+type Status struct {
+	AppName         string // Heighliner app name
+	Cd              CdInfo
+	Apps            []ApplicationInfo
+	SCM             SCM
+	TfConfigMapName string
+}
+
+// CdInfo Cd info
+type CdInfo struct {
+	Provider string `json:"provider" yaml:"provider"`
+	URL      string `json:"url" yaml:"url"`
+	Username string `json:"username" yaml:"username"`
+	Password string `json:"password" yaml:"password"`
+}
+
+// ApplicationInfo app info
+type ApplicationInfo struct {
+	Name     string
```

Comment:
Add tags to these fields too? Some of the fields already have tags. It looks more elegant to keep all of them to have tags.